content/en/tracing/profiler/getting_started.md

```diff
@@ -256,6 +256,83 @@ The Datadog Profiler requires Go 1.12+. To begin profiling applications:
 [6]: /tracing/visualization/#services
 [7]: /tracing/guide/setting_primary_tags_to_scope/#environment
 {{< /programming-lang >}}
+{{< programming-lang lang="ruby" >}}
+
+<div class="alert alert-warning">
+Datadog Ruby Profiler is currently in public beta. We strongly advise evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+The Datadog Profiler requires Ruby 2.1+. To begin profiling applications:
+
+1. If you are already using Datadog, upgrade your agent to version [7.20.2][1]+ or [6.20.2][1]+.
```

Comment:
In the Python docs, we list that CPU-time profiling only works "on POSIX platforms". For Ruby, the CPU-time profiler only currently works on Linux -- do you think it's worth calling that out here?